main/settings.py

```diff
@@ -650,6 +650,27 @@
     required=False,
 )
 
+# Wayback Machine Settings
+ENABLE_WAYBACK_TASKS = get_bool(
```

Comment:
We should mention the Wayback related config in the readme

Reply:
Mentioned here: https://github.com/mitodl/ocw-studio/pull/2326